src/lksearch/TESSSearch.py

```diff
@@ -287,76 +289,26 @@ def _sort_TESS(self):
         )
         self.table = df
 
+    @deprecated(
```

Comment:
This only raises a warning, but the module can no longer return what the user wants. I think it should raise an error.